src/components/scenes/FioAddressSettingsScene.js

```diff
@@ -6,20 +6,20 @@ import * as React from 'react'
 import { refreshAllFioAddresses } from '../../actions/FioAddressActions.js'
 import s from '../../locales/strings'
 import { FioActionSubmit } from '../../modules/FioAddress/components/FioActionSubmit'
-import { getTransferFee } from '../../modules/FioAddress/util'
+import { addBundles, getAddBundlesFee, getTransferFee } from '../../modules/FioAddress/util'
```

Comment:
Please change all variables that have `bundles` to `bundledTxs` or `bundled_txs` bundles is not a proper name